client/user/src/mealorder/menu/meal-order-user-cart.html

```diff
@@ -0,0 +1,179 @@
+            }
+
+            .order-quantity {
+                width: 15%;
+            }
+
+            .amount-title {
+                width: 50%;
+                text-align: left;
+                padding-left: 4%;
+                font-weight: bolder;
+            }
+
+            .amount-order-price {
+                width: 20%;
+                text-align: right;
+
+            }
+
+            .total {
+                display: flex;
+                justify-content: space-between;
+                margin: 6% 4% 10% 4%;
+                font-weight: bolder;
+            }
+        </style>
+        <div class="cart">
+
+            <dom-repeat items="[[order]]" as="dish">
+                <template>
+                    <div class="vendor">
+                        <div>[[dish.vendor]]</div>
+                        <paper-icon-button icon="delete" class="icon-delete"></paper-icon-button>
+                    </div>
+                    <dom-repeat items="[[dish.dishes]]" as="d">
+                        <template>
+                            <div class="order-details">
+                                <div class="order-title">[[d.title]]</div>
+                                <div class="order-price">[[d.price]] ₴</div>
+                                <div class="order-quantity">[[d.quantity]]</div>
+                                <paper-icon-button icon="remove" on-click="_removeDish"></paper-icon-button>
+                            </div>
+                        </template>
+                    </dom-repeat>
+
+                    <div class="order-details">
+                        <div class="amount-title">Сумма:</div>
+                        <div class="amount-order-price">500 ₴</div>
+                        <div class="order-quantity"></div>
+                        <paper-icon-button></paper-icon-button>
+                    </div>
+                </template>
+            </dom-repeat>
+
+            <div class="total">
+                <div class="amount">Общая сумма:</div>
+                <div class="price">1400 ₴</div>
+            </div>
+
+        </div>
+    </template>
+    <script>
+        /**
+         * user-cart is user's order for current date.
+         *
+         * <p>Is adopted for both desktop and mobile.
+         *
+         * @customElement
+         * @polymer
+         */
+        class MealOrderUserCart extends Polymer.Element {
+            static get is() {
+                return 'meal-order-user-cart';
+            }
+
+            static get properties() {
+                return {
+                    /**
+                     * Fired on dish remove button tap.
+                     *
+                     * @event `dish-removed`
+                     * @param {dish} contains vendor, title.
+                     */
+
+                    /**
+                     * Fired when a dish add button is tapped.
+                     *
+                     * @event `order-removed`
```

Comment:
`order-removed` is fired when add button is tapped?
What's the difference between `dish-removed` and `order-removed`?